recipe/meta.yaml

```diff
@@ -1,43 +1,50 @@
 {% set name = "hist" %}
 {% set version = "2.8.0" %}
+{% set python_min = "3.8" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hist-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hist-{{ version }}.tar.gz
   sha256: 0a3e602dd1d2721bd7f2229f456709dde323f6f74952f13ba4e5986c3275f77b
 
 build:
-  number: 1
+  number: 2
 outputs:
   - name: hist-base
     build:
       noarch: python
       entry_points:
         - hist=hist.classichist:main
-      script: python -m pip install . --no-deps -vv
+      # c.f. https://github.com/conda/grayskull/issues/582#issuecomment-2482370940
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
Hm. I guess `{{ PYTHON }}` won't work here, even though this is `noarch: python`. Maybe it is related to why there are multiple Python builds?